src/components/Sponsors/Sponsors.js

```diff
@@ -72,7 +72,7 @@ export function Sponsors () {
         >
           <img
             alt="eleven labs"
-            src={withPrefix('/images/sponsor/eleven-labs.png')}
+            src={withPrefix('/images/sponsor/eleven-labs.svg')}
```

Comment:
```suggestion
            src={withPrefix('/images/sponsor/eleven-labs.png')}
```

le logo supprimé était le bon, pas besoin de le modifier